packages/TelegramClientTests-UI.package/TCTUMessageTests.class/instance/testTextAndPhotoScaleCorrectlyLongCaption.st

```diff
@@ -0,0 +1,9 @@
+testing
+testTextAndPhotoScaleCorrectlyLongCaption
+
+	| photoMessage |
+
+	photoMessage := TCTMMocks mockPhotoMessageWith: self core.
+	photoMessage caption: 'This is a very very very very veeeeeeeeeeery long caption with many irregularly sized words. It even consists of two sentences and definitely bothers the Linter.'.
```

Comment:
if the linter is bothered, please use `"@linter-ignore"` on line after method name

Reply:
No linter tests fail